mb_bulk_copy_work_codes.user.js

```diff
@@ -579,16 +579,43 @@ function storeData(source, iswcs, codes, title) {
     GM_setValue('workCodeData', JSON.stringify(obj));
 }
 
+let translateStrings = (function() {
+    let strings;
+    const stringsDefaults = {
+        AGENCY_NAME_FIELD: 'Agency Name',
+        AGENCY_WORK_CODES: 'Agency Work Codes',
+        AGENCY_WORK_CODE_FIELD: 'Agency Work Code',
+        ARCHIVED_ISWCS: 'Archived ISWCs',
+        ORIGINAL_TITLE_FIELD: 'Original Title',
+        PREFERRED_ISWC_FIELD: 'Preferred ISWC',
+    };
```

Comment:
These are the strings which used to be hard-coded. We're using these as defaults in case `strings` isn't present in local storage for any reason whatsoever.